include/fluent-bit/flb_input_log.h

```diff
@@ -27,6 +27,11 @@ int flb_input_log_append(struct flb_input_instance *ins,
                          const char *tag, size_t tag_len,
                          const void *buf, size_t buf_size);
 
+int flb_input_log_append_ex(struct flb_input_instance *ins,
```

Comment:
_ex suffix is enough to distinguish from the not needed an argument for starting stage of processor.
However, this prefix is too general to show what represent to show how it works this API.
How about using more specific suffix name such as **_stage** or **_processor_stage**?

This is because other API uses **_records** suffix for distinguishing how it works.

Reply:
I agree with you that `_ex` is a bad suffix for the case but couldn't really come up with a short yet meaningful name.
Maybe `flb_input_log_append_at_processor_stage` or  `flb_input_log_append_skip_processor_stages` would be clear enough but they are painfully long.

I'll change it for `flb_input_log_append_skip_processor_stages` which even though it's ridiculously long clearly conveys the intention. 

Thanks for making me go over it again!